src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState(moviesFromServer);
+
+  const changeMoviesList = (searchQuery: string) => ([...moviesFromServer]
```

Comment:
```suggestion
  const changeMoviesList = (searchQuery: string) => (moviesFromServer
```
Фильтр делает не мутирует массив